parser/prism/Translator.cc

```diff
@@ -328,6 +328,42 @@ std::unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::Kwrestarg>(parser.translateLocation(loc), gs.enterNameUTF8(name));
         }
+        case PM_LOCAL_VARIABLE_AND_WRITE_NODE: {
```

Comment:
Does it make any sense to try to DRY some of this up?

Reply:
Definitely, but I'm not yet sure how. This will repeat a ton more as we repeat this for ivars, class vars, global vars, constants, constant paths, index calls and method calls.